### PR TITLE
Update realtime-log-analytics.md

### PR DESCRIPTION
**Pull request by nobuyoshi-macrometa**
**Changes**
- Update demo description. This is from new repository README.md
- Update Setup section's table 
- Remove `fastly*` suffix from Macrometa resource names
- Update repository URL in Solution section (step 4) and GitHub section (last one)

### CODE REVIEW

**Review by jyuwei — APPROVED** [review]

File: docs/apps/realtime-log-analytics.md [hunk 1]
```diff
@@ -3,38 +3,32 @@ sidebar_position: 4
 title: Realtime Log Analytics
 ---
 
-Realtime Log Analytics integrating logs from Fastly with Macrometa GDN to monitor the status and activity of Stream workers and Query workers. Fastly provides data about HTTP latency, response count, response size, and unique visitor traffic.
-
+This demo shows how to integrate real-time log analytics with Macrometa GDN to monitor the status and activity of Stream workers and Query workers.
```
Comment (diana-macrometa) [inline docs/apps/realtime-log-analytics.md]:
```suggestion
This demo shows how to integrate real-time log analytics with Macrometa GDN to monitor the status and activity of stream workers and query workers.
```

File: docs/apps/realtime-log-analytics.md [hunk 2]
```diff
@@ -3,38 +3,32 @@ sidebar_position: 4
...
 
 ## Setup
 
 | **Federation**                                        | **Email**                              | **Passsword** | **Dashboard**|
```
Comment (diana-macrometa) [inline docs/apps/realtime-log-analytics.md:10]: We need to replace "Federation" with something else in all cases.

File: docs/apps/realtime-log-analytics.md [hunk 3]
```diff
@@ -3,38 +3,32 @@ sidebar_position: 4
...
 | ----------------------------------------------------- | -------------------------------------- | ------------- |--------------|
-| [Global Data Network](https://gdn.paas.macrometa.io/) | demo-fastly-realtime-logs@macrometa.io | `xxxxxxxx`    | [Dashboard](https://macrometacorp.github.io/demo-fastly-jsc8-realtime-logs) |
-| [Fastly Account](https://manage.fastly.com)           | demo@macrometa.com                     | --            | -- |
-
-
+| [GDN](https://gdn.paas.macrometa.io/) | demo-realtime-logs@macrometa.io | `xxxxxxxx`    | [Dashboard](https://macrometacorp.github.io/demo-jsc8-realtime-logs) |
```
Comment (diana-macrometa) [inline docs/apps/realtime-log-analytics.md:12]: What value do the first three columns here provide to the end user?

File: docs/apps/realtime-log-analytics.md [hunk 4]
```diff
@@ -3,38 +3,32 @@ sidebar_position: 4
...
 
 ## Solution
```
Comment (diana-macrometa) [inline docs/apps/realtime-log-analytics.md:14]: This is the wrong heading. What follows are instructions or a procedure. Try replacing this with something more descriptive and adding a short introduction explaining to the user what they are going to do next. Provide context.

File: docs/apps/realtime-log-analytics.md [hunk 5]
```diff
@@ -3,38 +3,32 @@ sidebar_position: 4
...
 
-
 1. Create and publish the following Stream Workers in your GDN account:
```
Comment (diana-macrometa) [inline docs/apps/realtime-log-analytics.md]:
```suggestion
1. Create and publish the following stream workers in your GDN account:
```

File: docs/apps/realtime-log-analytics.md [hunk 6]
```diff
@@ -3,38 +3,32 @@ sidebar_position: 4
...
-fastly-http-request-worker
-fastly-http-request-stats-1m-worker
+log-generator
+http-request-worker
+http-request-stats-1m-worker
 ```
 
-
-2. Create the following Query workers in your GDN account:
+2. Create the following Query Workers in your GDN account:
```
Comment (diana-macrometa) [inline docs/apps/realtime-log-analytics.md]:
```suggestion
2. Create the following query workers in your GDN account:
```

**Review by diana-macrometa — CHANGES_REQUESTED** [review]
Thanks for the edits! I have some questions and suggestions.

File: docs/apps/realtime-log-analytics.md [hunk 7]
```diff
@@ -43,23 +37,22 @@ fastlyGetUniqueVisitorsByCountry
...
 
 4. On the development machine, run the following commands in a console:
 
 ```
-1. git clone git@github.com:Macrometacorp/demo-fastly-jsc8-realtime-logs.git
-2. cd fastly-edgeworker-log-analytics
-3. git fetch
-4. npm install
-5. npm run start
+1. git clone git@github.com:Macrometacorp/demo-jsc8-realtime-logs.git
```
Comment (diana-macrometa) [inline docs/apps/realtime-log-analytics.md]: What are these numbers doing here? Won't they cause an error if I copy and paste the code block in the terminal?

File: docs/apps/realtime-log-analytics.md (hunk 3 above)
Comment (jyuwei) [reply docs/apps/realtime-log-analytics.md:12]: It's for a quick reference of demo tenant on GDN and UI front

**Comment by jyuwei** [comment]
@diana-macrometa - Thank you for the feedback, these changes are needed for the Akamai demo, we will probably revert the changes later. But we will apply the changes you suggested afterwards.

**Review by diana-macrometa — APPROVED** [review]
I can live with that. I would love to partner with the Solutions team to improve all the Demos.

**Comment by jyuwei** [comment]
@diana-macrometa - Absolutely. I will invite you to the solutions daily meeting, so we can have some sessions to work on existing demos. Thank you!

**Pull request merged**